SmartDeviceLink/TestSubscribeButtonObserver.m

```diff
@@ -0,0 +1,66 @@
+//
+//  TestSubscribeButtonObserver.m
+//  SmartDeviceLink
+//
+//  Created by Nicole on 6/22/20.
+//  Copyright © 2020 smartdevicelink. All rights reserved.
+//
+
+#import "TestSubscribeButtonObserver.h"
```

Comment:
A warning is appearing here: "Method definition for 'buttonPressEventWithButtonName:error:buttonPress:buttonEvent:extraParameter:' not found"

Reply:
Fixed. Added